scripts/build/opensim-gui-linux-build-script.sh

```diff
@@ -190,10 +208,24 @@ echo
 echo "LOG: BUILDING OPENSIM-CORE..."
 mkdir -p ~/opensim-workspace/opensim-core-build || true
 cd ~/opensim-workspace/opensim-core-build
-cmake ~/opensim-workspace/opensim-core-source -G"$GENERATOR" -DOPENSIM_DEPENDENCIES_DIR=~/opensim-workspace/opensim-core-dependencies-install/ -DBUILD_JAVA_WRAPPING=on -DBUILD_PYTHON_WRAPPING=on -DOPENSIM_C3D_PARSER=ezc3d -DBUILD_TESTING=off -DCMAKE_INSTALL_PREFIX=~/opensim-core -DOPENSIM_INSTALL_UNIX_FHS=off -DSWIG_DIR=~/swig/share/swig -DSWIG_EXECUTABLE=~/swig/bin/swig
+cmake ~/opensim-workspace/opensim-core-source -G"$GENERATOR" -DOPENSIM_DEPENDENCIES_DIR=~/opensim-workspace/opensim-core-dependencies-install/ -DBUILD_JAVA_WRAPPING=on -DBUILD_PYTHON_WRAPPING=on -DOPENSIM_C3D_PARSER=ezc3d -DBUILD_TESTING=off -DCMAKE_INSTALL_PREFIX=~/opensim-core -DOPENSIM_INSTALL_UNIX_FHS=off -DSWIG_DIR=~/swig/share/swig -DSWIG_EXECUTABLE=~/swig/bin/swig -DOPENSIM_WITH_CASADI=$MOCO -DOPENSIM_WITH_TROPTER=$MOCO
 cmake . -LAH
 cmake --build . --config $DEBUG_TYPE -j$NUM_JOBS
+echo
+
+# Test opensim-core.
+echo "LOG: TESTING OPENSIM-CORE..."
+cd ~/opensim-workspace/opensim-core-build
+# TODO: Temporary for python to find Simbody libraries.
+export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:~/opensim-workspace/opensim-core-dependencies-install/simbody/lib
+ctest --parallel $NUM_JOBS --output-on-failure
+
+# Install opensim-core.
+echo "LOG: INSTALL OPENSIM-CORE..."
+cd ~/opensim-workspace/opensim-core-build
 cmake --install .
+echo 'export PATH=~/opensim-core/bin:$PATH' >> ~/.bashrc
```

Comment:
I'd recommend invoking the opensim install script instead.

Reply:
Hmmm actually I'm not sure that whole block (lines 223-229) is necessary. The opensim-core build outputs are copied/installed with the OpenSim GUI by default, and the bash script at https://github.com/opensim-org/opensim-gui/blob/220fe774fa69aa4090f2c5eb20d661e71d499ab8/scripts/build/opensim-gui-linux-build-script.sh#L251 calls the `./opensim-install-command-line.sh` script that installs a link to `opensim-cmd` in `/usr/local/bin`. I think as it stands, this build/install script is double installing opensim-core (one copy at the default install prefix, and another copy within the opensim GUI directory).

Reply:
That could be the case. I haven't had a look into that INSTALL script yet and won't be able to until Friday. Generally I hope this PR could be used to get some attention on the overdue refactor of the linux build scripts.